client/src/components/commonComponents/BackButton/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import './index.css';
+
+const BackButton = ({ back }) => (
+  <div
+    role="button"
+    tabIndex={0}
+    onClick={back}
+    onKeyDown={e =>
+      e.key === 'Backspace' || e.key === 'Escape' ? back() : null
```

Comment:
just FYI, this will only work when the element has the focus. If you want to catch a key pressed every time, it gets a bit more complicated:
https://stackoverflow.com/questions/37440408/how-to-detect-esc-key-press-in-react-and-how-to-handle-it/46123962
(up to you if you wanna look into that, I would personally leave it for now and focus on more important things)

Reply:
yup i felt that when i tested it, but eslint was making a fuss so i had to add it